soc/renesas/rcar/rcar_gen3/Kconfig

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2021 IoT.bzh
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RCAR_GEN3_R7
+	bool
+	select ARM
+	select CPU_CORTEX_R7
+	select PLATFORM_SPECIFIC_INIT
+	select CPU_HAS_DCLS
+	select CLOCK_CONTROL_RCAR_CPG_MSSR if CLOCK_CONTROL
+
+config SOC_SERIES_RCAR_GEN3_A57
+	bool
+	select ARM64
+	select CPU_CORTEX_A57
+	select SOC_FAMILY_RCAR
```

Comment:
as far as I see, you've renamed this SoC family

Reply:
Removed